pkgs/applications/misc/brewtarget/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib,
+mkDerivation,
+fetchFromGitHub,
+bash,
+cmake,
+wrapQtAppsHook,
+qtbase,
+qttools,
+qtmultimedia,
+qtwebkit,
+qtsvg
+}:
+
+let
+  pname = "brewtarget";
```

Comment:
does it not accept `pname` and `version`?

maybe you're missing the `rec` in `mkDerivation rec { ...` so we can self reference.